ethereum/spec/src/main/resources/tech/pegasys/teku/spec/config/configs/chiado.yaml

```diff
@@ -169,8 +104,8 @@ GOSSIP_MAX_SIZE: 10485760
 MAX_REQUEST_BLOCKS: 1024
 # `2**8` (= 256)
 EPOCHS_PER_SUBNET_SUBSCRIPTION: 256
-## `MIN_VALIDATOR_WITHDRAWABILITY_DELAY + CHURN_LIMIT_QUOTIENT // 2`
-MIN_EPOCHS_FOR_BLOCK_REQUESTS: 2304
+# `MIN_VALIDATOR_WITHDRAWABILITY_DELAY + CHURN_LIMIT_QUOTIENT // 2` (= 33024, ~5 months)
```

Comment:
```suggestion
# `MIN_VALIDATOR_WITHDRAWABILITY_DELAY + CHURN_LIMIT_QUOTIENT // 2` (= 33024, ~31 days)
```

Reply:
possibly missing the point that this is copied and pasted from chiado config?

Reply:
I can change this but its not goingto be right next time we copy it...